cmd/task-schedular-server/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"strconv"
+
+	loads "github.com/go-openapi/loads"
+	"github.com/spf13/viper"
+
+	runtime "github.com/AliShan302/Task_Schedular"
+	"github.com/AliShan302/Task_Schedular/config"
+	"github.com/AliShan302/Task_Schedular/gen/restapi"
+	"github.com/AliShan302/Task_Schedular/handlers"
+)
+
+func main() {
+	swaggerSpec, err := loads.Embedded(restapi.SwaggerJSON, restapi.FlatSwaggerJSON)
+	if err != nil {
+		panic(err)
+	}
+
+	rt, err := runtime.NewRuntime()
+	if err != nil {
+		panic(err)
+	}
+
+	api := handlers.NewHandler(rt, swaggerSpec)
```

Comment:
why are you not passing context.TODO() here?
we should create a ctx variable.

Reply:
ok shahzad bhai will do that